model_compression_toolkit/gptq/keras/quantization_facade.py

```diff
@@ -59,7 +59,7 @@
     def get_keras_gptq_config(n_iter: int,
                               optimizer: OptimizerV2 = tf.keras.optimizers.Adam(learning_rate=LR_DEFAULT),
                               optimizer_rest: OptimizerV2 = tf.keras.optimizers.Adam(learning_rate=LR_REST_DEFAULT),
-                              loss: Callable = GPTQMultipleTensorsLoss(),
+                              loss: Callable = GPTQMultipleTensorsLoss(norm_loss=False),
```

Comment:
Maybe we should change the default value in GPTQMultipleTensorsLoss init method to False?